spacy/lang/hi/stop_words.py

```diff
@@ -0,0 +1,177 @@
+# coding: utf8
+from __future__ import unicode_literals
+
+
+# Source: https://github.com/taranjeet/hindi-tokenizer/blob/master/stopwords.txt
+
+STOP_WORDS = set("""
+अत
+अपना
+अपनी
+अपने
+अभी
+अंदर
+आदि
+आप
+इत्यादि
+इन
+इनका
+इन्हीं
+इन्हें
+इन्हों
+इस
+इसका
+इसकी
+इसके
+इसमें
+इसी
+इसे
+उन
+उनका
+उनकी
+उनके
+उनको
+उन्हीं
+उन्हें
+उन्हों
+उस
+उसके
+उसी
+उसे
+एक
+एवं
+एस
+ऐसे
+और
+कई
+कर
+करता
+करते
+करना
+करने
+करें
+कहते
+कहा
+का
```

Comment:
Also add 'अर्थात', 'किन्तु', 'परन्तु', 'अपितु', 'कदाचित'  ???  - need a Hindi Pandit to ratify ?

Reply:
Isn't 'कदाचित' Marathi word, I have almost never seen it used in Hindi.

Reply:
Neh, its in Hindi, Marathi and Sanskrit (check ??).  U definitely see its wide usage in periodic movies and epic hindi serials.

Reply:
.. but i could be wrong too - i would say let's keep that out of stopwords - if some Hindi literary guy could comment on its frequency of usage and if it deserves to be a StopWord or not - let's go with that

Reply:
Actually the base script used is 'Devanagri' (https://en.wikipedia.org/wiki/Devanagari) so its a valid word in both languages (Hindi, Marathi), Also not sure what qualifies as a valid token in NLP is it just the occurrence or the usage in the actual language? I wish someone with Hindi Literature understanding would take a look that would be helpful.

Reply:
+!, hence my earlier comment about a Hindi Pandit having to ratify this.

Reply:
Stop words don't have to be 100% perfect, but in general (also for tokenizer exceptions and other language data), we always try to make sure spaCy also covers the *actual* usage in the real world. For example, in the English stop words, including the word "thru" is almost more important and relevant than the word "whence" (which is usually pretty uncommon in the type of text people are using with spaCy). So keeping a good balance here is always good 👍

Reply:
That's a great point @ines, i think we are good then with what we have here. +1 to merge.

Reply:
Great – merging this now so it's on `develop` and can be tested! If anyone wants to contribute more data or tests, you can submit a PR to the `develop` branch 😊  [See here](https://alpha.spacy.io/usage/adding-languages) for the usage guide on adding language data to spaCy in v2.0.